multicore_runtime/wsgi.py

```diff
@@ -43,6 +43,19 @@
 
 root_logger.setLevel(logging.INFO)
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
+  import googleclouddebugger
+  googleclouddebugger.AttachDebugger()
+except BaseException:
```

Comment:
Things that are wrong other than an issue with the debugger can generate exceptions at any time. Let's narrow it down to ImportError, especially since that would be consistent with the changes to the python27 runtime.